Assets/_Project/Code/Scripts/Leaderboard/Leaderboardscript.cs

```diff
@@ -0,0 +1,70 @@
+using System.Collections.Generic;
+using System.IO;
+using UnityEngine;
+using TMPro;
+using UnityEngine.UI;
+using System;
+
+[System.Serializable]
+public class TempTeam : IComparable<TempTeam>
```

Comment:
To do podmiany na aktualną klase druzyny, szczegolnie ze unity wypluwa 'TempTeam' is missing the class attribute 'ExtensionOfNativeClass'!